com.krioltech.fnmap.yml

```diff
@@ -0,0 +1,48 @@
+app-id: com.krioltech.fnmap
```

Comment:
is this controlled by the upstream developers?

I get `404 Not Found`

Reply:
The web site is https://www.krioltech.com

Reply:
![image](https://github.com/flathub/flathub/assets/114441/55217537-3761-4ad8-8c9b-df3e66cdc8a1)

Reply:
without www
![image](https://github.com/flathub/flathub/assets/114441/60698f9f-9b2a-418f-9ade-c062ac594ab0)

Reply:
Should work now. https://www.krioltech.com

Reply:
I'm getting a bad gateway still

Reply:
Ok.  Please try again.